### ISSUE TITLE
cumulative layout shift

### ISSUE THREAD

**Issue by felixroos** [mortuosplango/topfenstrudel#6]
when the component becomes active, the iframe pops in, changing the height of the parent element, shifting the layout. this is a bit annoying because the scroll position jumps around (especially when scrolling up). suggested fix: inactive component should already show a placeholder box of the same size

**Comment by mortuosplango** [comment]
Good idea. I don't know when I'll get around to doing that. Feel free to put up a PR if you're bored.
